### DISCUSSION TITLE
Add 'date' field for movies

### DISCUSSION THREAD

**Discussion by GreenSnakeLinux** [iptv-org/community#999] [Ideas]
I have parsed different XML files from various sites, and here it is missing a field very important for me: the 'date' field for movies
It is a common field handled by [epg xml_tv dtd ](https://github.com/XMLTV/xmltv/blob/master/xmltv.dtd)
Generally there is only the year of the movie released in cinema in this field, others categories does not have this field.

thanks to add it

**Comment by PopeyeTheSai10r** [comment]
This will depend on the source. If the source provide the information, they can be able to added, 
if the source doesn't have the information, there's nothing that can be done.

**Comment by GreenSnakeLinux** [comment]
I know but I have try every source of French TV, there is not this field.
When I go to the site of one of the source: https://www.programme-tv.net
the information is available, so I guess the field is missing in the parser, can you look at it ?

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/epg/issues/2307
